ERCS/erc-7824.md

```diff
@@ -0,0 +1,332 @@
+---
+eip: 7824
+title: Nostr and NFT Compatible Social Media Standard
+author: Nick Juntilla <nick@ownerfy.com>
+discussions-to: https://ethereum-magicians.org/t/eip-7832-nostr-and-nft-compatible-social-media-publishing-standard/22280
+status: Draft
+type: Standards Track
+category: ERC
+discussions: https://ethereum-magicians.org/t/eip-7832-nostr-and-nft-compatible-social-media-publishing-standard/22280
```

Comment:
```suggestion
discussions: https://ethereum-magicians.org/t/erc-7847-nostr-and-nft-compatible-social-media-publishing/22280
```
Updated Eth Magicians title with assigned number.
Discussions topic could just include a link to this PR and the abstract.  You shouldn't copy/paste the entire ERC as it will diverge through the editing process.